serdect/Cargo.toml

```diff
@@ -30,6 +30,7 @@ serde = { version = "1.0.119", default-features = false, features = ["derive"] }
 serde_json = "1"
 serde-json-core = { version = "0.5", default-features = false, features = ["std"] }
 toml = "0.7"
+rmp-serde = "1"
```

Comment:
Could you relocate this a couple lines above to preserve the ordering?

Reply:
Done